lucene/common-build.xml

```diff
@@ -764,7 +764,7 @@
       <!-- If possible, include the GIT hash into manifest: -->
       <exec dir="." executable="${git.exe}" outputproperty="-checkoutid" errorproperty="-giterr" failifexecutionfails="false">
         <arg value="log"/>
-        <arg value="--format=%H"/>
+        <arg value="--format=%h"/>
```

Comment:
I think it could also be a more verbose command like this one (note the extended abbreviated hash syntax):
```
git show --pretty=oneline --abbrev-commit --abbrev=15 --no-patch HEAD
```